game/enemies/enemy.py

```diff
@@ -14,7 +17,7 @@
 class Enemy(pygame.sprite.Sprite):
     def __init__(self, spritesheet, colisionHandler, wander_time: int, frame_width:int, frame_height:int,
                  num_frames, x, y, speed, attack_type, attack_damage, attack_range, health, colision_group,
-                 sprites_group, enemy_group, enemy_type='default', scale=1, player=None, projectile_path=None, projectile_cooldown=0, type="enemy"):
+                 sprites_group, enemy_group, enemy_type='default', scale=1, player : Player=None, projectile_path=None, projectile_cooldown=0, type="enemy"):
```

Comment:
If you are going to add types, add them to all parameters

Reply:
I fixed it by removing the type for player